tensorflow/python/ops/math_ops.py

```diff
@@ -3305,6 +3305,15 @@ def log_sigmoid(x, name=None):
   Specifically, `y = log(1 / (1 + exp(-x)))`.  For numerical stability,
   we use `y = -tf.nn.softplus(-x)`.
 
+  Usage Example:
+   >>>import tensorflow as tf
+   >>>tensor1 = [0.0, 1.0, 2.0, 3.0, 4.0, 5.0]
+   >>>tensor2 = [2.0, 5.7]
+   >>>print(tf.math.log_sigmoid(tensor1, name='log_sigmoid_tensor1'))
+   Tensor("log_sigmoid_tensor1_5:0", shape=(6,), dtype=float32)
```

Comment:
Make sure you run this in TF 2.0 as this is the output for TF 1.0. Then, you can make changes accordingly.